src/main/java/io/specto/hoverfly/junit/dsl/RequestedServiceBuilder.java

```diff
@@ -0,0 +1,21 @@
+package io.specto.hoverfly.junit.dsl;
+
+import io.specto.hoverfly.junit.dsl.matchers.PlainTextFieldMatcher;
+
+public class RequestedServiceBuilder extends AbstractServiceBuilder {
```

Comment:
yes, you probably right. So user will need to call `.anyParams()` or `anyBody()` explicitly to do fuzzy verification.